tools/viur-2to3.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
```

Comment:
This should not be part of the core (there is no need for it to be deployed). Also it's probably better to extend the existing 2to3 tool: http://python3porting.com/fixers.html

Reply:
Tools should be put into one place, for a fast access and central maintenance. If you want to use 2to3, feel free to integrate the changes there, I don't have time for this right now.

Reply:
I agree with both of you.
Generally I would prefer to have this (and other helping stuff like examples, unittests and co.) in the same repo.
But the core will not be build or shipped as pip-package, it's integrated as submodule. Therefore it's nice to keep it small as possible.

Maybe it's better placed in our [viur-tools repo](https://github.com/viur-framework/viur-tools).

Reply:
OK well, I don't want to have any more discussion on such little things here anymore, I have to get things done.
I'll gonna put the file into the viur-tools repo and I'm done with it.